src/components/home/index.tsx

```diff
@@ -0,0 +1,693 @@
+import React, {FunctionComponent} from 'react'
+import Card from './components/card'
+import EggheadPlayer from 'components/EggheadPlayer'
+import Link from 'next/link'
+import Image from 'next/image'
+import {map, get, find} from 'lodash'
+import Textfit from 'react-textfit'
+import Markdown from 'react-markdown'
+
+type HomeProps = {}
+
+const Home: FunctionComponent<HomeProps> = () => {
+  const data = [
```

Comment:
this is naive at best, but hopefully fine for initial pass? I haven't put much thought into whole authoring experience...

Reply:
i think this “brute force data glob” is a perfect place to start and will let us make a more considered authoring experience in the future. I dig it.

Reply:
I also wrote this tiny transform function that takes a url and returns data in a format that is expected here. 
saved me a little bit of time: https://codesandbox.io/s/quick-data-transformer-brjd3